src/index.js

```
@@ -128,4 +134,68 @@ process.on('SIGINT', async () => {
+
+// Run in all environments with different schedules
+const CLEANUP_INTERVAL = process.env.NODE_ENV === 'production'
+  ? 12 * 60 * 60 * 1000  // 12 hours in production
+  : 1  * 60 * 60 * 1000; // 1 hour in development
+
+const FILE_AGE_LIMIT = process.env.NODE_ENV === 'production'
+  ? 60 * 60 * 1000   // 1 hour in production
+  : 15 * 60 * 1000;  // 15 minutes in development
+
+if (true) {
+  // Clean temp files older than the configured age on the configured interval
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)